editor/store/selectors.js

```diff
@@ -1744,3 +1744,21 @@ export function getSupportedBlocks( state, uid, globallyEnabledBlockTypes ) {
 export function getEditorSettings( state ) {
 	return state.settings;
 }
+
+/*
+ * Returns the editor settings.
+ *
+ * @param {Object} state Editor state.
+ *
+ * @return {Object} The editor settings object
+ */
+export function getLocking( state, rootUID ) {
```

Comment:
I see this function being called using `getLocking( rootUID )` a few times. Am I misunderstanding something?

Reply:
The state is automatically injected by the data module

Reply:
(see the README here https://github.com/WordPress/gutenberg/tree/master/data)

Reply:
Oh, okay. That is really hard to grok from reading the code 😢